javascript/sails/config/routes.js

```diff
@@ -0,0 +1,5 @@
+module.exports.routes = {
+  "GET /": { action: "home" },
+  "GET /user/:id?": { action: "user/get" },
```

Comment:
is this (having `?`) means that `/user` will be allowed in `GET` ?

Reply:
Yes, you're right.
I've removed this.